src/lib/homestore/homeobject.cpp

```diff
@@ -1,14 +1,60 @@
 #include "homeobject.hpp"
 
+#include <homestore/homestore.hpp>
+#include <homestore/index_service.hpp>
+#include <iomgr/io_environment.hpp>
+
 namespace homeobject {
 
 extern std::shared_ptr< HomeObject > init_homeobject(std::weak_ptr< HomeObjectApplication >&& application) {
     LOGINFOMOD(homeobject, "Initializing HomeObject");
     auto instance = std::make_shared< HSHomeObject >(std::move(application));
+    instance->init_homestore();
     instance->init_repl_svc();
     return instance;
 }
 
+///
+// Start HomeStore based on the options retrived from the Application
+//
+// This should assert if we can not initialize HomeStore.
+//
+void HSHomeObject::init_homestore() {
+    auto app = _application.lock();
+    RELEASE_ASSERT(app, "HomeObjectApplication lifetime unexpected!");
+
+    LOGINFO("Starting iomgr with {} threads, spdk: {}", app->threads(), false);
+    ioenvironment.with_iomgr(iomgr::iomgr_params{.num_threads = app->threads(), .is_spdk = app->spdk_mode()});
+
+    /// TODO Where should this come from?
+    const uint64_t app_mem_size = 2 * Gi;
+    LOGINFO("Initialize and start HomeStore with app_mem_size = {}", homestore::in_bytes(app_mem_size));
```

Comment:
we usually do like this in POD spec,  that can copy any field in POD spec into ENV and make it available inside container.  Application can read from system env via `std::getenv()`.

In this case I believe we want to have `AVAIL_MEM_SIZE` set to POD's memory limit and take certain percentage from that for  each component. Maytbe same thing for the #thread? 

But these logic should better stay in SM? and pass in via APP?      
```
    env:
    - name: MINIO_GATEWAY_ACCOUNT_NAME
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: metadata.labels['nuobject-account']
         
```

Reply:
```
    - name: POD_IP
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: status.podIP
```